mobile/src/features/Notifications/useCases/ViewNotificationHistory/ViewNotificationHistoryScreen.tsx

```diff
@@ -70,7 +70,7 @@ export const ViewNotificationHistoryScreen = () => {
 
   return (
     <SafeAreaView
-      style={[a.px_lg, a.flex, {flex: 1, position: 'relative'}]}
+      style={[a.flex_1, a.px_lg]}
       edges={['right', 'left', 'bottom']}
     >
       <ScrollView contentContainerStyle={{paddingBottom: 60}}>
```

Comment:
if u have a ScrollView the padding horizontal should not stay in the main window, otherwise the scroll indicator will look off the right side, move the px to contentContainerStyle and to the other views, u can test it with many notifications, with no scrollable view is not a problem, but after many children it will look off